### DISCUSSION TITLE
Can a users email address be changes similar to passwords with OTP

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#20605] [Questions] [answered]
Hi,  Can a users email address be changed without using a link to a webpage, as this project is a mobile app.... can it be done with OTP (6 digit token) like with passwords?

**Comment by GaryAustin1** [comment, marked as answer]
You should be able to use verifyOtp.
https://supabase.com/docs/reference/javascript/auth-verifyotp
It mentions email_change.
